sketch/Explorer_ESP_WiFi-localization_v1.2.ino

```diff
@@ -0,0 +1,162 @@
+#include <TheThingsNetwork.h>
+
+#define HEX_CHAR_TO_NIBBLE(c) ((c >= 'a') ? (c - 'a' + 0x0A) : (c - '0'))
+#define HEX_PAIR_TO_BYTE(h, l) ((HEX_CHAR_TO_NIBBLE(h) << 4) + HEX_CHAR_TO_NIBBLE(l))
+
+// Parameters for the esp stream/buffer
+#define SF_ESP8266_DEFAULT_WAIT 5000
+#define SF_ESP8266_BUFFER_SIZE 128
+#define ESP_SERIAL_SPEED 9600
+#define ACCESS_POINTS 3
+#define SF_WIFI_BSSID_SIZE 6
```

Comment:
`SF_` was sensor fleet; rename

Reply:
Also in general; inconsistent uppercase defines vs lowercase defines in this file